app/Http/Controllers/Rest/CampaignsController.php

```diff
@@ -29,10 +30,11 @@ class CampaignsController extends AppController
 {
     public function add(Request $request)
     {
-//        $this->validateRequestObject($request, 'campaign', Campaign::$rules);
+        $this->validateRequestObject($request, 'campaign', Campaign::$rules);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "campaign" 9 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1192)